type-tests/files/createAction.typetest.ts

```diff
@@ -110,22 +115,21 @@ function expectType<T>(p: T): T {
 }
 
 /*
- * Test: createAction() type parameter is optional (defaults to `any`).
+ * Test: createAction() type parameter is required, not inferred (defaults to `void`).
```

Comment:
this test doesn't really make any more sense (description does not match what it tests right now), so I would delete it

Reply:
yeah, lemme rewrite this one